packages/flutter/lib/src/material/app.dart

```diff
@@ -163,9 +166,41 @@ class MaterialApp extends StatefulWidget {
   /// This value is passed unmodified to [WidgetsApp.onGenerateTitle].
   final GenerateAppTitle onGenerateTitle;
 
-  /// The colors to use for the application's widgets.
+  /// The [ThemeData] to use when the platform's requested brightness is [Brightness.light] or undefined.
```

Comment:
We try and keep doc comment lines to < 80 characters.

Although it's helpful to expand on the original one-line of information about this property, it seems to have become a little too focused its relationship to darkTheme etc.

How about this:
```
Default visual properties, like colors fonts and shapes, for this app's material widgets.

A second [darkTheme] [ThemeData] value, which is used when the underlying platform requests a "dark mode" UI, can also be specified.

The default value of this property is [ThemeData.light()].
```

Reply:
I changed it to your recommendation.